# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/renderer/d3d/TextureD3D.cpp
# Commit: 264003682fbf
# Full Hash: 264003682fbf25a121deeeb305d206a7cd437b94
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2019-12-27 09:44:18
# Description:
#   Bug 1601703 - Cherry-pick lost no-error context fix for GetProgramiv. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D58287
# ==============================================================================

diff -r 3888d3abdc8a -r 264003682fbf gfx/angle/checkout/src/libANGLE/renderer/d3d/TextureD3D.cpp
--- a/gfx/angle/checkout/src/libANGLE/renderer/d3d/TextureD3D.cpp	Tue Dec 10 05:28:58 2019 +0000
+++ b/gfx/angle/checkout/src/libANGLE/renderer/d3d/TextureD3D.cpp	Fri Dec 27 05:43:03 2019 +0000
@@ -205,7 +205,8 @@
 
     if (!mRenderer->getFeatures().setDataFasterThanImageUploadOn128bitFormats.enabled)
     {
-        gl::InternalFormat internalFormat = gl::GetSizedInternalFormatInfo(getBaseLevelInternalFormat());
+        gl::InternalFormat internalFormat =
+            gl::GetSizedInternalFormatInfo(getBaseLevelInternalFormat());
         return internalFormat.pixelBytes < 16;
     }
 